doc/interfaces/bundlewarp_registration_flow.rst

```diff
@@ -34,22 +34,22 @@ moving bundle using a single regularization parameter, alpha (represented with
 λ in BundleWarp paper). Where alpha controls the trade-off between regularizing
 the deformation and having points match very closely. The lower the value of
 alpha, the more closely the bundles would match. Here, we investigate how to
-warp moving bundle with different levels of deformations using BundleWarp
+warp a moving bundle with different levels of deformations using BundleWarp
 registration method [Chandio23]_.
 
 --------------------------------------------
 Partially Deformable BundleWarp Registration
 --------------------------------------------
 
-Here, we partially deform/warp moving bundle to align it with static bundle.
+Here, we partially deform/warp the moving bundle to align it with the static bundle.
 partial deformations improve linear registration while preserving the anatomical
-shape and structures of the moving bundle. Here, we use relatively higher value
-of alpha=0.03. By default, BundleWarp partially deforms the bundle to preserve
+shape and structures of the moving bundle. Here, we use a relatively higher value
+of alpha=0.05. By default, BundleWarp partially deforms the bundle to preserve
```

Comment:
```suggestion
of alpha=0.5. By default, BundleWarp partially deforms the bundle to preserve
```

Reply:
Hi @arokem 

There is a subtle difference between registrations with alpha=0.3 and alpha=0.5, both resulting in fewer deformations. In our paper, we present results using both values (see [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9881938/)). However, please note that this preprint is not updated and does not include some recent experiments.

Here is one of the figures from paper demonstrating effect of different alpha values on registration:

![image](https://github.com/dipy/dipy/assets/26060471/5b9af5b2-ee0d-4e33-b8ce-962aaec1abaf)

And here is another example comparing results visually and using shape similarity score as quantitative metric with alpha=0.5 and alpha=0.3:
-------------------
**Static**
![image](https://github.com/dipy/dipy/assets/26060471/12c55bf6-df4b-49ac-bb24-6caae7d566d9")
-------------------
**Affinely moved bundle** 
Shape similarity score between static and affinely moved bundle  =  0.513
<img src="https://github.com/dipy/dipy/assets/26060471/2f0250e5-630f-41c8-831d-0c26c08401a8" width=94% height=59%>
-------------------
**Registered with alpha=0.5**
Shape similarity score between static and moved bundle  = 0.688
<img src="https://github.com/dipy/dipy/assets/26060471/79ab39f8-15a6-47d3-befd-045c2511d354" width=96% height=50%>
-------------------
**Registered with alpha=0.3**
Shape similarity score between static and moved bundle  = 0.697
<img src="https://github.com/dipy/dipy/assets/26060471/3facd1de-b0bb-4662-947a-3ae745593c7f" width=110% height=50%>
-------------------

I've noticed some discrepancies between the partially deformable registration example in the API tutorial and the CLI tutorial. Specifically, the API tutorial uses an alpha value of 0.5 (see [API tutorial](https://docs.dipy.org/stable/examples_built/registration/bundlewarp_registration.html#sphx-glr-examples-built-registration-bundlewarp-registration-py)), while the CLI tutorial uses 0.3 but mistakenly lists it as 0.03 (see [CLI tutorial](https://docs.dipy.org/stable/interfaces/bundlewarp_registration_flow.html)). There are also a few typos and mistakes in the documentation.

I propose setting alpha to the highest recommended default value of 0.5 and correcting the typos in the examples. Using alpha=0.5 ensures that the registration process remains strict, avoiding heavy deformations. Users can always decrease the value if they wish to introduce more deformation to the bundles.

Reply:
Yeah - that all sounds very reasonable to me. From the visuals at least, it seems that the difference between 0.3 and 0.5 is rather minor, but seems reasonable to default to 0.5. Thanks for all the details!